front-end/src/pages/AuthorPage.tsx

```diff
@@ -48,15 +48,15 @@ export default function AuthorPage(props: any) {
     if (props.loggedInUser) {
       // get whether user is follower of author IF not looking at our own profile
       if (!props.location.pathname.includes(props.loggedInUser.authorId)) {
-        AxiosWrapper.get(authorUrl + "/followers/" + props.loggedInUser.authorId).then((res: any) => {
+        AxiosWrapper.get(authorUrl + "/followers/" + props.loggedInUser.authorId, props.loggedInUser).then((res: any) => {
```

Comment:
can you add this to have a trailing slash?

Reply:
Changed